applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -277,7 +277,7 @@ def __init__(self, identifier, response_settings, model):
 
         self.primal_analysis = StructuralMechanicsAnalysis(model, primal_parameters)
 
-        self.use_python_primal_solution_data_transfer = self._UsePythonPrimalDataTransfer()
+        self.use_python_primal_solution_data_transfer = self.response_settings["primal_data_transfer_with_python"].GetBool() 
```

Comment:
give the variable the same name as the setting: primal_data_transfer_with_python